test/forge/HalmosTest.sol

```diff
@@ -49,8 +44,7 @@ contract HalmosTest is SymTest, Test {
         marketParams = MarketParams(address(loanToken), address(collateralToken), address(oracle), address(irm), lltv);
         id = marketParams.id();
 
-        vm.assume(block.timestamp != 0);
```

Comment:
in the end, this is not needed?

Reply:
I changed the function `check_lastUpdateNonZero` to verify for an arbitrary `id`. To do that I needed to assume the precondition of the invariant which makes the line you are highlighting useless in the end. We still rely on the fact that markets start with non zero timestamp, but it's not explicitely written in the verification